guake/guake_app.py

```diff
@@ -1042,8 +1043,24 @@ def recompute_tabs_titles(self):
             page_num = self.get_notebook().page_num(terminal.get_parent())
             self.get_notebook().rename_page(page_num, self.compute_tab_title(terminal), False)
 
+    def read_current_guake_yaml(self, vte):
+
+        cwd = Path(vte.get_current_directory())
+        guake_yaml = cwd.joinpath(".guake.yml")
+        if guake_yaml.is_file():
+            with guake_yaml.open() as fd:
```

Comment:
Open should have explicit file encoding specified.

Reply:
I added the encoding